spec/index.spec.js

```diff
@@ -58,6 +58,26 @@ describe('axiosRetry(axios, { retries })', () => {
     }, done.fail);
   });
 
+  it('should resolve with a succesful response after an error with custom retries', done => {
```

Comment:
As I said in my previous comment, I'd rewrite this as `should use the retries option of the request config`